test/new-e2e/pkg/e2e/suite.go

```diff
@@ -517,7 +517,13 @@ func (bs *BaseSuite[Env]) TearDownSuite() {
 // Unfortunately, we cannot use `s Suite[Env]` as Go is not able to match it with a struct
 // However it's able to verify the same constraint on T
 func Run[Env any, T Suite[Env]](t *testing.T, s T, options ...SuiteOption) {
-	options = append(options, WithDevMode())
+	devMode, err := runner.GetProfile().ParamStore().GetBoolWithDefault(parameters.DevMode, false)
```

Comment:
I forgot that you would need to implement the param read from config file as well here: https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/pkg/runner/parameters/store_config_file.go

Reply:
I don't think we need to expose DevMode in the configuration file as it's will only used in local setup
or is this mandatory ?

Reply:
Not mandatory but handy: I use the local config file to set common parameters as it is more straightforward than using env variables with VsCode.